adapters/Vungle/VungleAdapter/GADMAdapterVungleRewardedAd.m

```diff
@@ -35,6 +35,10 @@ @implementation GADMAdapterVungleRewardedAd {
   BOOL _isRewardedAdPresenting;
 }
 
++ (NSString *)adapterVersion {
+    return kGADMAdapterVungleVersion;
+}
```

Comment:
What is this for? It doesn't look like it is used anywhere?

Can we remove this and use the `+version:` from `GADMediationAdapterVungle`, or just use `kGADMAdapterVungleVersion` directly if this is needed?

Reply:
@joshuajplagonera Hi joshuajplagonera, the class method `adapterVersion` is only used from our QA Test app to display adapter version number in log. So I removed this method from `GADMAdapterVungleInterstitial` and `GADMAdapterVungleRewardedAd` class. Thanks very much for your suggestions.🙂